src/rtFileDownloader.h

```diff
@@ -79,6 +81,8 @@ class rtFileDownloadRequest
 #ifdef ENABLE_HTTP_CACHE
   void setCacheEnabled(bool val);
   bool cacheEnabled();
+  void setDataCache(bool val);
+  bool dataCached();
```

Comment:
Can you please rename setDataCache() to setDataIsCached() and dataCache() to isDataCached()?